eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -211,7 +211,7 @@ jobs:
   
   - ${{ if eq(parameters.targetOS, 'windows') }}:
     - script: |
-        call $(sourcesPath)\build.cmd -ci -cleanWhileBuilding -prepareMachine
+        call $(sourcesPath)\build.cmd -ci -cleanWhileBuilding -prepareMachine ${{ parameters.extraProperties }}
```

Comment:
i think this should also be added to the non-windows invocations.

Reply:
The non-Windows invocations already pass this along